app/main.py

```diff
@@ -191,49 +191,49 @@ def root(request: Request):
     """
     Display a welcome message and a link to the API documentation.
     """
-    return """
+    return f"""
```

Comment:
:cherries: rather than make it all an f-string, which then means we have to escape the `{` stuff, why not make just the link line f-string and assemble it?